modules/ROOT/pages/test-specification-api-console.adoc

```diff
@@ -0,0 +1,32 @@
+= Test your API Specification Using the API Console
+
+Studio provides an API Console view for the API you are designing. The API Console view returns the HTTP status codes and example payloads that you defined in your API specification, and is valuable for testing or exploring how your API behaves.
+
+Simulating calls to your API is a critical design task that helps you troubleshoot problems and demo the API to prospective users even before you have implemented it. You can simulate calls to the API in the API Console view before publishing a new version of the API specification to Exchange.
+
+When you activate the API Console view it exposes a mocking service that you can use for testing as if it was an actual implementation of you API specification.
+
+== Before You Begin
+
+* You must have an API project created.
+* Your API project must be synchronized with Design Center.
+
+== Procedure
+
+. Locate the API Console view within the API Design perspective. +
+The API Console view displays:
+* A summary of your API specificaiton.
+* All the available endpoints along with their allowed HTTP methods.
+* A list of all the types defined in your API specifcaition.
+. Navigate through the the *Endpoints* section and click on the HTTP method that you want to test.
+. Select the *Try It* button to simulate calls to an API. +
```

Comment:
```suggestion
. Click *Try It* to simulate calls to an API. +
```
Style Ref:
UI elements
Refer to UI elements by label rather than element type.

Don’t specify the following UI element types unless you need to for clarity or navigation: 

button		checkbox	 field			icon		
menu		link		 radio button 		window

	Before	3. Select the Upload all my files now checkbox and click the Next button.
	After		3. Select Upload all my files now and click Next.